cmd/rook/ceph/operator.go

```diff
@@ -91,14 +111,132 @@ func startOperator(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		rook.TerminateFatal(err)
 	}
-
 	rookImage := rook.GetOperatorImage(context.Clientset, containerName)
 	serviceAccountName := rook.GetOperatorServiceAccount(context.Clientset)
 	op := operator.New(context, volumeAttachment, rookImage, serviceAccountName)
+	if isPresent, err := isSecretPresent(context); isPresent && err == nil {
+		err = initWebhook(context)
+		if err != nil {
+			logger.Errorf("an error has occurred while initializing webhook: %+v", err)
+		}
+	}
+	if err != nil {
+		logger.Errorf("error has occurred: %+v", err)
```

Comment:
Add more context here. Error while doing what?

Reply:
ditto, `failed to ...` and use `%v`

Reply:
fixed...